### DISCUSSION TITLE
[Doc] should this be the other way around?

### DISCUSSION THREAD

**Discussion by Alex1989Wang** [pmndrs/jotai#2178] [General]
From this link: https://jotai.org/docs/core/use-atom

<img width="935" alt="image" src="https://github.com/pmndrs/jotai/assets/4620388/2f86b579-df27-480a-993b-06135109cd43">

Shouldn't it be "*uppercaseAtom* has a dependency on *textAtom*"?

**Comment by dai-shi** [comment]
Yeah, it should mean the same as "uppercaseAtom depends on textAtom". We could remove the read underline sentence entirely.
@sandren your thoughts?
